modules/llama_cpp_plugin/src/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set( PLUGIN_LIBRARY_NAME CACHE STRING "Library name for the generated plugin" ${TARGET_NAME})
+if(NOT PLUGIN_LIBRARY_NAME)
+    set( PLUGIN_LIBRARY_NAME "llama_cpp_plugin" )
+endif()
+
+set( PLUGIN_DEVICE_NAME CACHE STRING "Device name for the resulting plugin")
+if(NOT PLUGIN_DEVICE_NAME)
+    set( PLUGIN_DEVICE_NAME "LLAMA_CPP" )
+endif()
+
+set(TARGET_NAME ${PLUGIN_LIBRARY_NAME})
+
+file(GLOB_RECURSE SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
+file(GLOB_RECURSE HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
+
+if (NOT ENABLE_LLAMA_CPP_PLUGIN_REGISTRATION)
+    # Skip install and registration of template component
+    set(skip_plugin SKIP_INSTALL SKIP_REGISTRATION)
+endif()
+
+
+
+# adds a shared library with plugin
+ov_add_plugin(NAME ${TARGET_NAME}
+              DEVICE_NAME ${PLUGIN_DEVICE_NAME}
+              SOURCES ${SOURCES} ${HEADERS}
+              ${skip_plugin}
+              VERSION_DEFINES_FOR plugin.cpp
+              ADD_CLANG_FORMAT)
+
+# Enable support of CC for the plugin
+ov_mark_target_as_cc(${TARGET_NAME})
```

Comment:
I suppose CC is not supported for this plugin and we can drop it

Reply:
Done